resources/js/views/settings/Settings.vue

```diff
@@ -1,76 +1,22 @@
 <template>
   <b-container class='mt-3 mb-5'>
     <h2>{{ $t('settings.title') }}</h2>
+    <b-button
```

Comment:
Could we put the home button on the same line with the title?

Reply:
Would you prefer left or right of the titel?

Reply:
I would prefer left of the title!